legal-copy/terms-of-use.md

```diff
@@ -15,10 +15,10 @@ May 3, 2016
     Any feedback you share with us will only be made publicly accessible in an aggregated format without identifying you.
 
 0. ## Privacy
-    [The Test Pilot Privacy Notice](https://testpilot.firefox.com/privacy) describes, generally, what data we receive from your participation in any Test Pilot Experiment. We use the information we receive through Test Pilot as described in our [Mozilla Privacy Policy](https://www.mozilla.org/privacy/).
+    [The Test Pilot Privacy Notice](/privacy) describes, generally, what data we receive from your participation in any Test Pilot Experiment. We use the information we receive through Test Pilot as described in our [Mozilla Privacy Policy](https://www.mozilla.org/privacy/).
```

Comment:
The "Privacy" fully-qualified URL was bugging me since I was verifying dev or stage, clicked the link, and was accidentally verifying on prod now.
This URL should work a-OK for us, but may break upstream if anybody is displaying our legal docs elsewhere (which they aren't until I get around to #738).